docs/rules/no-native-attributes.md

```diff
@@ -1,7 +1,7 @@
 # Disallows use of native attributes as properties (no-native-attributes)
 
 Using global native attributes as lit properties can have unintended effects,
-like for example the native `title` attribute will cause a tiny popup to show up
+like for example the native `title` attribute will display a tooltip on hover
 over your custom element, and may mess up the accessibility tree for your
```

Comment:
bit of a nitpick but also `and may affect the accessibility of your component unintentionally`